tests/test_dbos.py

```diff
@@ -36,6 +27,66 @@ def test_communicator(var: str) -> str:
         DBOS.logger.info("I'm test_communicator")
         return var
 
+    @dbos.workflow()
```

Comment:
Could the child workflow tests be split into a different test than the simple workflow tests?

Reply:
Of course they could, it would just be a lot of cut-n-paste.  I actually started with a separate file and then thought this was very fundamental and may as well stay in the base test.  If the file is too big I would nominate recovery to be split out, not child workflow.

Reply:
I was thinking different test in the same file, not different file, just to better localize errors.

Reply:
It'd mean cutting+pasting the counters and tx/communicator functions but will if you want, it'll get a lot better with class and method decorators  :-D

Reply:
Yeah, let's separate, it's good to have some dead-simple tests even if it means some redundancy.